### DISCUSSION TITLE
Issues running the installer

### DISCUSSION THREAD

**Discussion by jareddewitt** [ihsn/nada#72] [Q&A]
I'm having issues running the installer after setup.  When I browse to **http://your-domain/nada-folder-name**, i get redirected to **http://your-domain/nada-folder-name/index.php/install** and only a blank pages is displayed. 

This seems to happen in docker as well as a fresh install on nginx, php7.3, mysql.

**Comment by mah0001** [comment]
To see the errors instead of a blank page, please try this and report back the errors:

Edit index.php file and on line 66 change the environment value from 'production' to 'development'. 

from:
```
define('ENVIRONMENT', isset($_SERVER['CI_ENV']) ? $_SERVER['CI_ENV'] : 'production');
```

change to:

```
define('ENVIRONMENT', isset($_SERVER['CI_ENV']) ? $_SERVER['CI_ENV'] : 'development');
```

**Reply by jareddewitt** [reply]
Thanks @mah0001 , this helped.  I was able to determine it was a mysql access issue.  

I'm still unable to run the installer though.  When I browse to my site, I'm redirected to "/index.php/install' and receive a 404 from nginx instead of the NADA installer page.

**Comment by mah0001** [comment]
To run the installer, you will need to have an empty database created. See if creating a database resolves the issue.
